examples/burgers_n2n/network.py

```diff
@@ -0,0 +1,43 @@
+from nn_adapt.layout import NetLayoutBase
```

Comment:
Similarly, you could plausibly import from `examples/burgers/network.py`, unless you plan to change the network architecture.